dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -57,4 +57,19 @@
       https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/407
     </Reason>
   </LinkageError>
+  <LinkageError>
+    <Target>
+      <Package name="com.sun.jna" />
+    </Target>
+    <Source>
+      <Package name="reactor.blockhound.shaded.net.bytebuddy.agent.VirtualMachine" />
+    </Source>
+    <Reason>
+      BlockHound shades ByteBuddy's VirtualMachine class. The class uses Java Native Access (JNA)
+      only when the com.sun.jna package is available by catching ClassNotFoundException. Therefore
```

Comment:
This sentence doesn't quite work. Perhaps break out " by catching ClassNotFoundException" into a sentence of its own that explains how the access is guarded, or just delete it.

Reply:
Right. Deleted "by catching ...".